src/status_im/data_store/mailservers.cljs

```diff
@@ -14,6 +16,71 @@
                                                  (core/get-all :mailserver)
                                                  (core/all-clj :mailserver))))))
 
+(defn mailserver-request-gaps->rpc [{:keys [chat-id] :as gap}]
+  (-> gap
+      (assoc :chatId chat-id)
```

Comment:
we already used kebab-case a lot for these status-go api

Reply:
cool, I might just change that in next PR if that's ok, as already in develop this one